src/main/java/net/md_5/specialsource/RemappingClassAdapter.java

```diff
@@ -83,8 +83,11 @@ public RemappingClassAdapter(final ClassVisitor cv, final CustomRemapper remappe
     @Override
     public MethodVisitor visitMethod(int access, String name, String desc,
             String signature, String[] exceptions) {
+        if (cv == null) {
```

Comment:
When can these be null?
It seems to me if this was null the whole class would not do anything anyway.
Should probably just be preconditioned in constructor

Reply:
Removed null checks and added to constructor as suggested